From 856c9a7506cb86bedeb1402f03266b5b6fb2c431 Mon Sep 17 00:00:00 2001
From: Joel Brobecker <brobecker@adacore.com>
Date: Fri, 7 Oct 2016 10:27:09 -0700
Subject: [PATCH 002/122] Bump GDB version number to 7.12.0.DATE-git.

gdb/ChangeLog:

	* version.in: Set GDB version number to 7.12.0.DATE-git.
---
 gdb/ChangeLog  | 4 ++++
 gdb/version.in | 2 +-
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/gdb/ChangeLog b/gdb/ChangeLog
index 9c18a5c63a..e03d5447ab 100644
--- a/gdb/ChangeLog
+++ b/gdb/ChangeLog
@@ -1,5 +1,9 @@
 2016-10-07  Joel Brobecker  <brobecker@adacore.com>
 
+	* version.in: Set GDB version number to 7.12.0.DATE-git.
+
+2016-10-07  Joel Brobecker  <brobecker@adacore.com>
+
 	GDB 7.12 released.
 
 2016-10-07  Joel Brobecker  <brobecker@adacore.com>
diff --git a/gdb/version.in b/gdb/version.in
index 1e0d11a4c1..20b9abb4e3 100644
--- a/gdb/version.in
+++ b/gdb/version.in
@@ -1 +1 @@
-7.12
+7.12.0.DATE-git
-- 
2.11.0

